README.md

```diff
@@ -60,34 +60,30 @@ You can also set the server name, but it is recommended to don't set this config
 ## Optional configurations
 
 ```yml
-# Not tested on Grails 3 plugin...
 grails:
     plugin:
         sentry:
-            dsn: https://foo:bar@api.sentry.io/123
+            dsn: https://foo@api.sentry.io/123
             loggers: [LOGGER1, LOGGER2, LOGGER3]
             environment: staging
             serverName: dev.server.com
-            levels: [ERROR]
+            minLevel: ERROR
             tags: {tag1: val1,  tag2: val2, tag3: val3}
-            subsystems: 
-                MODULE1: [com.company.services.module1, com.company.controllers.module1]
-                MODULE2: [com.company.services.module2, com.company.controllers.module2]
-                MODULE3: [com.company.services.module3, com.company.controllers.module3]
-            logClassName: true
             logHttpRequest: true
             disableMDCInsertingServletFilter: true
+            inAppIncludes:
+              - my.app.package.one
+              - my.app.package.two
+            tracesSampleRate: 0.5
```

Comment:
Is this a recommended default trace sample rate ? Should we add it the to the README ?

Reply:
No actually, just random example ;-)